File: SplitViewAppTemplate/Presentation/MenuItem.cs
using System;

namespace $safeprojectname$.Presentation
{
    public class MenuItem : NotifyPropertyChanged
    {
        private string icon;
        private string title;
        private Type pageType;


        public string Icon
        {
            get { return this.icon; }
            set { Set(ref this.icon, value); }
        }

        public string Title
        {
            get { return this.title; }
            set { Set(ref this.title, value); }
        }

        public Type PageType
        {
            get { return this.pageType; }
            set { Set(ref this.pageType, value); }
        }
    }
}

File: 0/Templates/VisualBasic/ProjectTemplates/SplitViewAppVBTemplate/Presentation/MenuItem.vb
Namespace Presentation
    Public Class MenuItem
        Inherits NotifyPropertyChanged

        Private _icon As String
        Private _title As String
        Private _pageType As Type

        Public Property Icon As String
            Get
                Return _icon
            End Get
            Set
                SetProperty(_icon, Value)
            End Set
        End Property

        Public Property Title As String
            Get
                Return _title
            End Get
            Set
                SetProperty(_title, Value)
            End Set
        End Property

        Public Property PageType As Type
            Get
                Return _pageType
            End Get
            Set
                SetProperty(_pageType, Value)
            End Set
        End Property

    End Class
End Namespace

